.clang-tidy

```diff
@@ -28,6 +28,20 @@ CheckOptions:
     value: '2'
   - key:   readability-identifier-length.MinimumVariableNameLength
     value: '2'
-  - key:   readability-identifier-length.MinimumParameterNameLength
-    value: '2'
```

Comment:
Did you mean to blow away the minimum parameter name length setting?

Reply:
No, copy-pasta error by idiot on keyboard :) Will fix momentarily.